src/js/components/Select/index.d.ts

```diff
@@ -32,10 +47,10 @@ export interface BasicSelectProps {
   margin?: MarginType;
   messages?: { multiple?: string };
   name?: string;
-  onChange?: (...args: any[]) => void;
-  onClose?: (...args: any[]) => any;
+  onChange?: (event: SelectOnChangeEvent) => void;
+  onClose?: React.MouseEventHandler<HTMLButtonElement>;
   onMore?: (...args: any[]) => any;
-  onOpen?: (...args: any[]) => any;
+  onOpen?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
I would expect `onClose` and `onOpen` to both be functions

Reply:
I checked the Type for the other Components, they have used similar Type for onOpen and onClose

Reply:
@TheThinker277 

I think this type if fine for the `onOpen` and `onClose`

React.MouseEventHandler<HTMLButtonElement> is a type that represents a function that handles mouse events specifically for a button.